util/oidc/oidc.go

```diff
@@ -234,19 +234,19 @@ func (a *ClientApp) HandleCallback(w http.ResponseWriter, r *http.Request) {
 	if a.secureCookie {
 		flags = append(flags, "Secure")
 	}
-	cookie, err := httputil.MakeCookieMetadata(common.AuthCookieName, idTokenRAW, flags...)
-	if err != nil {
-		http.Error(w, fmt.Sprintf("%v", err), http.StatusInternalServerError)
-		return
-	}
-	w.Header().Set("Set-Cookie", cookie)
-
 	var claims jwt.MapClaims
 	err = idToken.Claims(&claims)
 	if err != nil {
 		http.Error(w, err.Error(), http.StatusInternalServerError)
 		return
 	}
+	cookie, err := httputil.MakeCookieMetadata(common.AuthCookieName, idTokenRAW, flags...)
+	if err != nil {
+		http.Error(w, fmt.Sprintf("claims=%v, err=%v", claims, err), http.StatusInternalServerError)
```

Comment:
Can we move this new block after the `claimsJSON, _ := json.Marshal(claims)` line so that we can dump the actual claims in json format instead of a golang map format?